ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -731,4 +777,10 @@
         "OTHERS": GATHER_COMMANDS_OTHERS + GATHER_COMMANDS_OTHERS_4_6,
         "OTHERS_EXTERNAL": GATHER_COMMANDS_OTHERS_EXTERNAL_4_6,
     },
+    4.7: {
+        "CEPH": GATHER_COMMANDS_CEPH + GATHER_COMMANDS_CEPH_4_7,
+        "JSON": GATHER_COMMANDS_JSON + GATHER_COMMANDS_JSON_4_7,
+        "OTHERS": GATHER_COMMANDS_OTHERS + GATHER_COMMANDS_OTHERS_4_7,
+        "OTHERS_EXTERNAL": GATHER_COMMANDS_OTHERS_EXTERNAL_4_6,
```

Comment:
"_4_6", Should it be 4_7?
>>Should we change it for OTHERS_EXTERNAL too?

Reply:
@sidhant-agrawal

Reply:
apologies for the late reply.
We need to check if any new files added for Internal mode 4.7 is also valid for External mode 4.7
In External mode, they could have a slightly different name containing `external` keyword, or could be entirely absent.

If new files in 4.7 not applicable for external mode, then we can continue using 4.6 list